components/encryption/src/file_dict_file.rs

```diff
@@ -280,10 +288,11 @@ impl FileDictionaryFile {
 
     fn parse_next_record(mut remained: &[u8]) -> Result<(usize, String, LogRecord)> {
         if remained.len() < Self::RECORD_HEADER_SIZE {
-            return Err(box_err!(
-                "file corrupted! record header size is too small: {}",
-                remained.len()
-            ));
+            warn!(
+                "file corrupted! record header size is too small";
```

Comment:
can we mention this is the **tail** record that's corrupted (so it is probably fine to truncate it).